gradlew.bat

```diff
@@ -1,3 +1,22 @@
+@REM
+@REM Licensed to the Apache Software Foundation (ASF) under one
+@REM or more contributor license agreements. See the NOTICE file

```

Comment:
Hmm... are we changing the license that is on gradlew.bat? Surely we should retain the existing license that it is provided with

Reply:
That one was not intended... have to adjust the license excludes... fixing.

Reply:
Yup, as expected was the license plugin configuration that just included a bit too much (\*\*/\*.bat). Fixed.